src/secret-storage.ts

```diff
@@ -672,3 +672,30 @@ export function trimTrailingEquals(input: string): string {
         return input;
     }
 }
+
+/**
+ * Check whether the private keys exist in secret storage.
+ *
+ * @param secretStorage - The secret store using account data
+ * @returns map of key name to key info the secret is encrypted
+ *     with, or null if it is not present or not encrypted with a trusted
+ *     key
+ */
+export async function isStoredInSecretStorage(
```

Comment:
This needs a better name:

 * *What* is stored in secret storage?
 * It doesn't return a boolean, so we shouldn't call it `is...`.

Maybe `getCrossSigningKeysSecretStorageDetails` or something.

However...

given that all we need for the rust implementation here is a boolean, I would have left the legacy implementation alone and made a new function with a simpler interface.